### ISSUE TITLE
"stores at" return testing/garbage output

### ISSUE THREAD

**Issue by frafra** [Ephigenia/ikea-availability-checker#86]
**Describe the bug**
```
[frafra@e32 ~]$ npx ikea-availability-checker stores at | head -n5
npx: installed 20 in 1.466s
undefined
undefined
undefined
                                   !             H|H|H|H|H           H__________________________________             H|§|§|§|H           H|* * * * * *|---------------------|             H|§|∞|§|H           H| * * * * * |---------------------|             H|§|§|§|H           H|* * * * * *|---------------------|             H|H|H|H|H           H| * * * * * |---------------------|             H|H|H|H|H           H|---------------------------------|          ===============        H|---------------------------------|            /| _   _ |          H|---------------------------------|            (| O   O |)          H|---------------------------------|            /|   U   |          H-----------------------------------             |  =/  |           H              _..._/            H              _|I/|_            H      _______/| H |/_______    H     /           / /          H    |          | | /         |  H    |          ||o||          |  H    |    |     ||o||     |    |  H    |    |     ||o||     |    |  H   Carl Pilcher  
tͣ͂eͪͩsͫ̈tͤ̈ī̉̄n̏ͨg͊̈ͦ ̈̀ ҉͍̝̜t͐͒e͑̚s͌͒tͮ̊i̔̃ň̋g͆͗ ͦͣtͪͬe̿͌ ҉̠̳̼s̈̚tͤ̓iͬͣ̌n͋̎ ҉̙̜g̐ͤ ͫ́ ҉̜̼̕t̽̾ ҉̹̱͜e͌̇s̏̾ͣt̽͋iͬ̇ͯn̑͑g̓̿ ̀t̾͛eͭ͊s̾̅̌tͤ̇i͛̓ ҉̗̬͈͜͠n̂ͯg͋͊͛ ͨ͑t͌ͮe̐ͬs͒̚ ҉̶̧ ҉̗̟͎t͒̉ỉ͑̚ň̓g̍̈ ͯ̔t̓͐ ҉͚͘ė̑ ҉̪ͅs̓̑͑t̏̽iͮ͐n͌̆ǧ͒
^C[frafra@e32 ~]$ npx ikea-availability-checker -V
npx: installed 20 in 1.484s
1.0.5
```

**Comment by Ephigenia** [comment]
Ah sorry that is from the colors package I‘ll fix this tomorrow.
See https://github.com/Marak/colors.js/commit/074a0f8ed0c31c35d13d28632bd8a049ff136fb6

**Comment by Ephigenia** [comment]
@frafra should be fixed with the latest release 1.0.6

**Comment by frafra** [comment]
It works, thank you :)
